plexapi/library.py

```diff
@@ -2849,6 +2861,137 @@ def _loadData(self, data):
         self.type = data.attrib.get('type')
 
 
+class HubManage(PlexObject):
+    """ Represents a Managed Recommendation inside a library.
+
+        Attributes:
+            identifier (str): Identifier for the recommendation.
+            title (str): Title of recommendation.
+            recommendationsVisibility (str): Recommendation's visibility.
+            homeVisibility (str): Home's visibility.
+            promotedToRecommended (bool): Promoted to recommended.
+            promotedToOwnHome (bool): Promoted to own home.
+            promotedToSharedHome (bool): Promoted to shared home.
+            deletable (bool): Is the Hub deletable.
+    """
+
+    def _loadData(self, data):
+        """ Load attribute values from Plex XML response. """
+        self._data = data
+        self.identifier = data.attrib.get('identifier')
+        self.title = data.attrib.get('title')
+        self.recommendationsVisibility = data.attrib.get('recommendationsVisibility')
+        self.homeVisibility = data.attrib.get('homeVisibility')
+        self.promotedToRecommended = utils.cast(bool, data.attrib.get('promotedToRecommended', False))
+        self.promotedToOwnHome = utils.cast(bool, data.attrib.get('promotedToOwnHome', False))
+        self.promotedToSharedHome = utils.cast(bool, data.attrib.get('promotedToSharedHome', False))
+        self.deletable = utils.cast(bool, data.attrib.get('deletable'))
+        self._collection_key = None
+
+        parent = self._parent()
+        self._librarySectionID = parent.key if isinstance(parent, LibrarySection) else parent.librarySectionID
+
```

Comment:
Add `reload()` method.
```suggestion
    def reload(self):
        """ Reload the data for this managed hub. """
        key = f'/hubs/sections/{self.librarySectionID}/manage'
        hub = self.fetchItem(key, self.__class__, identifier=self.identifier)
        self.__dict__.update(hub.__dict__)
        return self

```